aspnetcore/mvc/models/model-binding/samples/6.0/ModelBindingSample/Snippets/ObjectIdConverter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace ModelBindingSample.Snippets;
+
+// <snippet_Class>
+internal class ObjectIdConverter : JsonConverter<ObjectId>
+{
+    public override ObjectId Read(
+            ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options) =>
```

Comment:
```suggestion
        ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options) =>
```

Reply:
And those are just suggestions. If you think the code looks better with the horizontal scroll bar, that's fine.